.gitignore

```diff
@@ -7,7 +8,8 @@ intra/split/example/example
 .LSOverride
 
 # Icon must end with two \r
-Icon

+Icon
+
```

Comment:
I think this lost the special carriage return

Reply:
I have no idea on how to make it not change. The editor keeps removing it. But do we even care about this file?

Reply:
This is part of Github's [recommended macOS .gitignore](https://github.com/github/gitignore/blob/main/Global/macOS.gitignore).  I don't know if it affects this project.

Reply:
Yeah, but VSCode insists on removing that ^M on save. I think preserving it is unnecessary trouble, since that file doesn't seem to matter anyway.

Reply:
In that case I think it would be better to remove this line and comment.

Reply:
Removed